app/Services/Validation/Rules/TwitterHandleRule.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Services\Validation\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+use Illuminate\Support\Facades\Lang;
+
+class TwitterHandleRule implements Rule
+{
+    /**
+     * Validation for Twitter handlers.
+     *
+     * @var string
+     */
+    const PATTERN = '^[A-Za-z0-9_]+$';
+
+    /**
+     *
+     * Twitters Terms of Service only allows ". A username can only contain
+     * alphanumeric characters (letters A-Z, numbers 0-9) with the exception
+     * of underscores"
+     * This regex will allow only alphamumeric characters and the underscore.
+     * Keep this handy if we need to validate other usernames.
+     *
+     * @param  string  $attribute
+     * @param  mixed   $value
+     * @return boolean
+     */
+    public function passes($attribute, $value)
+    {
+        return preg_match('/' . self::PATTERN . '/', $value);
```

Comment:
Since we know twitter handles have a max length, we should fold that into this rule as well.